src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -365,7 +365,7 @@ protected function doRequestInProcess($request)
             unlink($deprecationsFile);
             foreach ($deprecations ? unserialize($deprecations) : array() as $deprecation) {
                 if ($deprecation[0]) {
-                    trigger_error($deprecation[1], E_USER_DEPRECATED);
+                    @trigger_error($deprecation[1], E_USER_DEPRECATED);
```

Comment:
This one should be reverted

Reply:
I reverted it (your fabbot was asking to add the `@`)